docs/rules/no-restricted-imports.md

```diff
@@ -87,6 +99,22 @@ import cluster from 'cluster';
 import pick from 'lodash/pick';
 ```
 
+```js
+/*eslint no-restricted-imports: ["error", { paths: [{ name: "foo", objectNames: ["DisallowedObject"] }] }]*/
+
+import DisallowedObject from "foo"
```

Comment:
Are these examples backwards? Unless I'm misunderstanding what the rule is supposed to do, it seems like all of the examples in the "incorrect code" section are correct, and all of the examples in the "correct code" section are incorrect.

Reply:
This one is correct since DisallowedObject is just an alias for the default export.  If you want to exclude it, you would use ["default"]

If the code was import `{ DisallowedObject } from "foo"` then it would fail because DisallowedObject is the actual name of the import.

Reply:
This code block is under the header `Examples of **incorrect** code for this rule:`. Shouldn't it be under `Examples of **correct** code for this rule:` if it's correct?

Reply:
:facepalm: you are right!  I put them in backwards.  I'll update this, too.

Reply:
Updated in the 7th commit.